ql/src/java/org/apache/hadoop/hive/ql/exec/tez/CustomPartitionVertex.java

```diff
@@ -546,14 +534,13 @@ private Multimap<Integer, InputSplit> getBucketSplitMapForPath(String inputName,
     Multimap<Integer, InputSplit> bucketToSplitMap = ArrayListMultimap.create();
 
     boolean fallback = false;
-    for (Map.Entry<String, Set<FileSplit>> entry : pathFileSplitsMap.entrySet()) {
+    for (Map.Entry<Integer, Set<HiveInputSplit>> entry : bucketSplitsMap.entrySet()) {
       // Extract the buckedID from pathFilesMap, this is more accurate method,
       // however. it may not work in certain cases where buckets are named
       // after files used while loading data. In such case, fallback to old
       // potential inaccurate method.
       // The accepted file names are such as 000000_0, 000001_0_copy_1.
```

Comment:
could we please cleanup the comment since we no longer operate on fileNames

Reply:
Good point! Done
https://github.com/apache/hive/pull/5409/commits/ae7444e8e1c2de95bfc1915af3d3c0245af4b472